.github/workflows/cd-database-branching.yml

```diff
@@ -0,0 +1,46 @@
+name: Neon Branch Creation
+
+on:
+  push:
+    branches:
+      - 'prod'
+      - 'preprod'
+
+  workflow_dispatch:
```

Comment:
What does this trigger mean ?

Reply:
Here the cd will run automatically after any push on prod or preprod branch and the workflow dispatch will allow running manually the cd

Reply:
Great